### DISCUSSION TITLE
搜索卡死

### DISCUSSION THREAD

**Discussion by xxtyyq** [jxxghp/MoviePilot#2765] [问题咨询]
如图，从早上昨天晚上开始就卡在正在处理，不返显搜索结果了，求问这个是什么情况？清理缓存和容器重启都试过了![image](https://github.com/user-attachments/assets/cc75d4b0-e972-47b5-aeba-711f4de95d49)![image](https://github.com/user-attachments/assets/426f7c8f-52bc-4e45-84c3-a5586f9c245d)

**Comment by iamckun** [comment]
看看是不是磁盘io爆了，我经常这样只要磁盘一卡就用不起了
